workflows/Snakefile

```diff
@@ -126,4 +126,59 @@ rule run_mcmc_mark00:
         """
 
 
+## TO BE integrated in the above
```

Comment:
It'd be good to add `TODO` comment around this.

We use Google style guide, which recommends the format `TODO(Person who can be asked for more information): some description`, e.g., `TODO(Gordon): Use _config.py utilities to refactor this code`